common/core/web/keyboard-processor/src/text/kbdInterface.ts

```diff
@@ -948,20 +950,59 @@ namespace com.keyman.text {
       this.output(1, outputTarget, "");
     }
 
+    /**
+     * Function     processNewContextEvent
+     * Scope        Private
+     * @param       {Object}        outputTarget   The target receiving input
+     * @param       {Object}        keystroke      The input keystroke (with its properties) to be mapped by the keyboard.
+     * Description  Calls the keyboard's `begin newContext` group
+     * @returns     {number}        0 if no match is made, otherwise 1.
```

Comment:
Do these 3 processX() methods return `RuleBehavior`?

Or is `this.process()` returning `0` or `1`?

Reply:
Hmm, yes, the old comment on line 957(989) was wrong!